CHANGELOG.md

```diff
@@ -1,3 +1,81 @@
+# v0.31.0 Release - 06/06/2019
+
+New Features:
+* Add CI on windows [#2214](https://github.com/GoogleContainerTools/skaffold/pull/2214)
+* Add API build trigger [#2201](https://github.com/GoogleContainerTools/skaffold/pull/2201)
+* Cross compile Skaffold, with CGO=1, using xgo [#2006](https://github.com/GoogleContainerTools/skaffold/pull/2006)
+* Print logs from init containers [#2182](https://github.com/GoogleContainerTools/skaffold/pull/2182)
+
+Bug Fixes:
+* Restore defaulting to polling trigger [#2228](https://github.com/GoogleContainerTools/skaffold/pull/2228)
+* Fix master branch [#2221](https://github.com/GoogleContainerTools/skaffold/pull/2221)
+* Revert "Improve output for docker push" [#2216](https://github.com/GoogleContainerTools/skaffold/pull/2216)
+* Fix error in flag refactoring for `skaffold run --tail` [#2172](https://github.com/GoogleContainerTools/skaffold/pull/2172)
+
+Updates & refactoring:
+* Increase test coverage [#2225](https://github.com/GoogleContainerTools/skaffold/pull/2225)
+* Use test wrapper in more tests [#2222](https://github.com/GoogleContainerTools/skaffold/pull/2222)
+* Improve documentation for kaniko [#2186](https://github.com/GoogleContainerTools/skaffold/pull/2186)
+*  Use the testutils test helper [#2218](https://github.com/GoogleContainerTools/skaffold/pull/2218)
+* Remove AppVeyor [#2219](https://github.com/GoogleContainerTools/skaffold/pull/2219)
+* Check man page with a unit test instead of a script [#2180](https://github.com/GoogleContainerTools/skaffold/pull/2180)
+* Make notify trigger the default trigger [#2213](https://github.com/GoogleContainerTools/skaffold/pull/2213)
```

Comment:
This and "Restore defaulting to polling trigger" cancel each other.  :)

Reply:
I'll create a rolled back section!

Reply:
Hmm...if we have a create + rollback - we should just remove both!